examples/tabs/palette/src/components/SampleRouterComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, ControlGroup, Intent, H1, H3 } from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import { IRouterProvidedProps } from "@misk/simpleredux"
+import * as React from "react"
+import { Link } from "react-router-dom"
+import { CodePreContainer } from '@misk/core';
+
+const LinkComponent = (props: IRouterProvidedProps) => {
+  if (props.match) {
```

Comment:
Not sure I'm following the significance of this. if `props.match`, why would we swap to using an `a` link instead of `Link`?

Reply:
This is an unfortunate hack in place to pass tests that complain of a <Link> tag being used outside of a router.